src/kibana/plugins/vis_types/vislib/tile_map.js

```diff
@@ -13,11 +14,46 @@ define(function (require) {
       params: {
         defaults: {
           mapType: 'Scaled Circle Markers',
-          isDesaturated: true
+          isDesaturated: true,
+          autoPrecision: true
         },
         mapTypes: ['Scaled Circle Markers', 'Shaded Circle Markers', 'Shaded Geohash Grid'],
         editor: require('text!plugins/vis_types/vislib/editors/tile_map.html')
       },
+      listeners: {
+        mapZoomEnd: function (event) {
```

Comment:
This function never checks if autoPrecision is turned on, there's no way to disable it.